apps/braze/src/components/create/CreateStep.tsx

```diff
@@ -161,16 +183,28 @@ const ContentBlockView = ({ name, description, onEdit }: ContentBlockViewProps)
           </Stack>
         )}
       </Stack>
-      <IconButton
-        size="small"
-        variant="secondary"
-        icon={<PencilSimple size={16} />}
-        aria-label="Edit content block"
-        onClick={onEdit}
-        className={editButton}
-        withTooltip
-        tooltipProps={{ content: 'Edit name or add a description' }}
-      />
+      {isDisabled ? (
+        <IconButton
+          size="small"
+          variant="positive"
+          icon={<Check size={16} />}
```

Comment:
💭  I think we should use this one instead
```suggestion
          icon={<CheckCircle size={16} />}
```